.github/workflows/cli-tests.yml

```diff
@@ -0,0 +1,213 @@
+name: CLI Tests
+
+on:
+  workflow_call:
+    inputs:
+      snap-channel:
+        type: string
+        description: "Snap channel to install Multipass from"
+        required: false
+        default: "<undefined>"
+      macos-pkg-url:
+        type: string
+        description: "URL to macOS package to install Multipass from"
+        required: false
+        default: "<undefined>"
+      windows-pkg-url:
+        type: string
+        description: "URL to Windows package to install Multipass from"
+        required: false
+        default: "<undefined>"
+      pytest-extra-args:
+        type: string
+        description: "Extra arguments to be passed to the pytest invocation"
+        default: ""
+  workflow_dispatch:
```

Comment:
I think you could use YAML anchors to deduplicate the inputs.
https://github.blog/changelog/2025-09-18-actions-yaml-anchors-and-non-public-workflow-templates/

Reply:
Done